.github/workflows/benchmark.yml

```
@@ -18,17 +14,44 @@ jobs:
       - uses: actions/checkout@v3
```

actionlint (1.7.11)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)